test/l.jl

```diff
@@ -177,4 +181,138 @@
         end
     end
     kclear()
+
+    let stringtest = "one two three four"
+        exp = ["one", "two", "three", "four"]
+        act = lparse(stringtest, " ", 25)
+        @testset for i in eachindex(act, exp)
+            @test act[i] == exp[i]
+        end
+    end
+
+    let stringtest = "  A number of words   separated   by spaces   "
+        exp = ["A", "number", "of", "words", "separated", "by", "spaces"]
+        act = lparsm(stringtest, " ", 20, 23)
+        @testset for i in eachindex(act, exp)
+            @test act[i] == exp[i]
+        end
+        act = lparsm(stringtest, " ", length(stringtest)+10)
+        @testset for i in eachindex(act, exp)
+            @test act[i] == exp[i]
+        end
+    end
+
+    let stringtest = "  A number of words   separated   by spaces.   "
+        delims = " ,."
+        act = lparss(stringtest, delims)
+        exp = ["", "A", "by", "number", "of", "separated", "spaces", "words"]
+        @testset for i in 1:length(exp)
+            @test act[i] == exp[i]
+        end
+    end
+    
+    kclear()
+    furnsh(
+        path(CORE, :lsk),
+        path(CORE, :pck),
+        path(CORE, :spk))
+    let et = str2et("21 march 2005")
+        lon = rad2deg(lspcn("EARTH", et, "NONE"))
+        @test lon ≈ 0.48153755894179384
+    end
+    kclear()
+    
+
+    let array = ["BOHR", "EINSTEIN", "FEYNMAN", "GALILEO", "NEWTON"]
+        @test lstlec("NEWTON", array) == 5
+        @test lstlec("EINSTEIN", array) == 2
+        @test lstlec("GALILEO", array) == 4
+        @test lstlec("Galileo", array) == 4
+        @test lstlec("BETHE", array) == 0
+    end
+
+    let array = [-2.0, -2.0, 0.0, 1.0, 1.0, 11.0]
+        @test lstled(-3.0, array) == 0
+        @test lstled(-2.0, array) == 2
+        @test lstled(0.0, array) == 3
+        @test lstled(1.0, array) == 5
+        @test lstled(11.1, array) == 6
+    end
+
+    # ERROR: some don't pass their test
+    let array = Int64[-2, -2, 0, 1, 1, 11]
+        @test lstlei(-3, array) == 0
+        @test lstlei(-2, array) == 2
+        @test lstlei(0, array) == 3
+        @test lstlei(1, array) == 5
+        @test lstlei(12, array) == 6
+    end
+
+    let array = ["BOHR", "EINSTEIN", "FEYNMAN", "GALILEO", "NEWTON"]
+        @test lstltc("NEWTON", array) == 4
+        @test lstltc("EINSTEIN", array) == 1
+        @test lstltc("GALILEO", array) == 3
+        @test lstltc("Galileo", array) == 4
+        @test lstltc("BETHE", array) == 0
+    end
+
+    let array = [-2.0, -2.0, 0.0, 1.0, 1.0, 11.0]
+        @test lstltd(-3.0, array) == 0
+        @test lstltd(-2.0, array) == 0
+        @test lstltd(0.0, array) == 2
+        @test lstltd(1.0, array) == 3
+        @test lstltd(11.1, array) == 6
+    end
+
+    # ERROR: some don't pass their test
+    let array = Int64[-2, -2, 0, 1, 1, 11]
```

Comment:
If all elements in the array are integers the `Int64` annotation is not necessary.